github.com/cockroachdb/cockroach/pkg/sql: (unknown) failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/e17938643d4c88da56bdb27f94c808fb2b0c02b6

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=true
TAGS=
GOFLAGS=-race
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=55563&tab=buildLog

```
go list -tags '' -f 'go test -v -race -tags '\'''\'' -ldflags '\'''\'' -i -c {{.ImportPath}} -o {{.Dir}}/stress.test && (cd {{.Dir}} && if [ -f stress.test ]; then stress -maxtime 15m -maxfails 1 -stderr ./stress.test -test.run '\''.'\'' -test.timeout 0 -test.v; fi)' github.com/cockroachdb/cockroach/pkg/sql | /bin/bash
github.com/cockroachdb/cockroach/pkg/cli/cliflags
github.com/cockroachdb/cockroach/pkg/build
github.com/cockroachdb/cockroach/pkg/util/syncutil
github.com/cockroachdb/cockroach/pkg/util/log/logflags
github.com/cockroachdb/cockroach/pkg/util/humanizeutil
github.com/cockroachdb/cockroach/pkg/util/uuid
github.com/cockroachdb/cockroach/pkg/util/duration
github.com/cockroachdb/cockroach/pkg/util/decimal
github.com/cockroachdb/cockroach/pkg/util/httputil
github.com/cockroachdb/cockroach/pkg/util/retry
github.com/cockroachdb/cockroach/pkg/internal/rsg/yacc
github.com/cockroachdb/cockroach/pkg/util/caller
github.com/cockroachdb/cockroach/pkg/sql/privilege
github.com/cockroachdb/cockroach/pkg/util/interval
github.com/cockroachdb/cockroach/pkg/util/shuffle
github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror
github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb
github.com/cockroachdb/cockroach/pkg/util/bufalloc
github.com/cockroachdb/cockroach/pkg/ui
github.com/cockroachdb/cockroach/pkg/util/sdnotify
github.com/cockroachdb/cockroach/pkg/util/protoutil
github.com/cockroachdb/cockroach/pkg/sql/pgbench
github.com/cockroachdb/cockroach/pkg/util/encoding
github.com/cockroachdb/cockroach/pkg/util/log
github.com/cockroachdb/cockroach/pkg/util/envutil
github.com/cockroachdb/cockroach/pkg/util/cache
github.com/cockroachdb/cockroach/pkg/util/timeutil
github.com/cockroachdb/cockroach/pkg/util/randutil
github.com/cockroachdb/cockroach/pkg/util/tracing
github.com/cockroachdb/cockroach/pkg/util/hlc
github.com/cockroachdb/cockroach/pkg/security
github.com/cockroachdb/cockroach/pkg/util/metric
github.com/cockroachdb/cockroach/pkg/util
github.com/cockroachdb/cockroach/pkg/util/leaktest
github.com/cockroachdb/cockroach/pkg/storage/engine/enginepb
github.com/cockroachdb/cockroach/pkg/security/securitytest
github.com/cockroachdb/cockroach/pkg/sql/mon
github.com/cockroachdb/cockroach/pkg/roachpb
github.com/cockroachdb/cockroach/pkg/util/stop
github.com/cockroachdb/cockroach/pkg/keys
github.com/cockroachdb/cockroach/pkg/sql/parser
github.com/cockroachdb/cockroach/pkg/ts/tspb
github.com/cockroachdb/cockroach/pkg/storage/storagebase
github.com/cockroachdb/cockroach/pkg/base
github.com/cockroachdb/cockroach/pkg/util/netutil
github.com/cockroachdb/cockroach/pkg/config
github.com/cockroachdb/cockroach/pkg/util/grpcutil
github.com/cockroachdb/cockroach/pkg/testutils/storageutils
github.com/cockroachdb/cockroach/pkg/gossip/resolver
github.com/cockroachdb/cockroach/pkg/internal/client
github.com/cockroachdb/cockroach/pkg/testutils
github.com/cockroachdb/cockroach/pkg/rpc
github.com/cockroachdb/cockroach/pkg/gossip
github.com/cockroachdb/cockroach/pkg/server/status
github.com/cockroachdb/cockroach/pkg/sql/sqlutil
github.com/cockroachdb/cockroach/pkg/kv
github.com/cockroachdb/cockroach/pkg/server/serverpb
github.com/cockroachdb/cockroach/pkg/internal/rsg
github.com/cockroachdb/cockroach/pkg/sql/sqlbase
github.com/cockroachdb/cockroach/pkg/testutils/sqlutils
github.com/cockroachdb/cockroach/pkg/testutils/serverutils
github.com/cockroachdb/cockroach/pkg/sql/distsql
github.com/cockroachdb/cockroach/pkg/storage/engine
github.com/cockroachdb/cockroach/pkg/sql
github.com/cockroachdb/cockroach/pkg/storage
github.com/cockroachdb/cockroach/pkg/ts
github.com/cockroachdb/cockroach/pkg/sql/pgwire
github.com/cockroachdb/cockroach/pkg/server
github.com/cockroachdb/cockroach/pkg/testutils/testcluster
github.com/cockroachdb/cockroach/pkg/cli/cliflags
github.com/cockroachdb/cockroach/pkg/util/syncutil
github.com/cockroachdb/cockroach/pkg/util/log/logflags
github.com/cockroachdb/cockroach/pkg/build
github.com/cockroachdb/cockroach/pkg/util/humanizeutil
github.com/cockroachdb/cockroach/pkg/util/uuid
github.com/cockroachdb/cockroach/pkg/util/duration
github.com/cockroachdb/cockroach/pkg/util/decimal
github.com/cockroachdb/cockroach/pkg/util/httputil
github.com/cockroachdb/cockroach/pkg/util/retry
github.com/cockroachdb/cockroach/pkg/sql/privilege
github.com/cockroachdb/cockroach/pkg/util/caller
github.com/cockroachdb/cockroach/pkg/util/interval
github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror
github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb
github.com/cockroachdb/cockroach/pkg/util/bufalloc
github.com/cockroachdb/cockroach/pkg/util/shuffle
github.com/cockroachdb/cockroach/pkg/internal/rsg/yacc
github.com/cockroachdb/cockroach/pkg/ui
github.com/cockroachdb/cockroach/pkg/util/sdnotify
github.com/cockroachdb/cockroach/pkg/util/protoutil
github.com/cockroachdb/cockroach/pkg/sql/pgbench
github.com/cockroachdb/cockroach/pkg/util/encoding
github.com/cockroachdb/cockroach/pkg/util/log
github.com/cockroachdb/cockroach/pkg/util/envutil
github.com/cockroachdb/cockroach/pkg/util/cache
github.com/cockroachdb/cockroach/pkg/util/timeutil
github.com/cockroachdb/cockroach/pkg/util/randutil
github.com/cockroachdb/cockroach/pkg/util/tracing
github.com/cockroachdb/cockroach/pkg/util/hlc
github.com/cockroachdb/cockroach/pkg/util
github.com/cockroachdb/cockroach/pkg/security
github.com/cockroachdb/cockroach/pkg/util/metric
github.com/cockroachdb/cockroach/pkg/util/leaktest
github.com/cockroachdb/cockroach/pkg/storage/engine/enginepb
github.com/cockroachdb/cockroach/pkg/security/securitytest
github.com/cockroachdb/cockroach/pkg/sql/mon
github.com/cockroachdb/cockroach/pkg/roachpb
github.com/cockroachdb/cockroach/pkg/util/stop
github.com/cockroachdb/cockroach/pkg/keys
github.com/cockroachdb/cockroach/pkg/sql/parser
github.com/cockroachdb/cockroach/pkg/ts/tspb
github.com/cockroachdb/cockroach/pkg/storage/storagebase
github.com/cockroachdb/cockroach/pkg/base
github.com/cockroachdb/cockroach/pkg/util/netutil
github.com/cockroachdb/cockroach/pkg/config
github.com/cockroachdb/cockroach/pkg/util/grpcutil
github.com/cockroachdb/cockroach/pkg/testutils/storageutils
github.com/cockroachdb/cockroach/pkg/gossip/resolver
github.com/cockroachdb/cockroach/pkg/internal/client
github.com/cockroachdb/cockroach/pkg/testutils
github.com/cockroachdb/cockroach/pkg/rpc
github.com/cockroachdb/cockroach/pkg/gossip
github.com/cockroachdb/cockroach/pkg/server/status
github.com/cockroachdb/cockroach/pkg/sql/sqlutil
github.com/cockroachdb/cockroach/pkg/kv
github.com/cockroachdb/cockroach/pkg/server/serverpb
github.com/cockroachdb/cockroach/pkg/sql/sqlbase
github.com/cockroachdb/cockroach/pkg/testutils/sqlutils
github.com/cockroachdb/cockroach/pkg/internal/rsg
github.com/cockroachdb/cockroach/pkg/testutils/serverutils
github.com/cockroachdb/cockroach/pkg/sql/distsql
github.com/cockroachdb/cockroach/pkg/storage/engine
github.com/cockroachdb/cockroach/pkg/sql
github.com/cockroachdb/cockroach/pkg/sql
github.com/cockroachdb/cockroach/pkg/storage
github.com/cockroachdb/cockroach/pkg/ts
github.com/cockroachdb/cockroach/pkg/sql/pgwire
github.com/cockroachdb/cockroach/pkg/server
github.com/cockroachdb/cockroach/pkg/testutils/testcluster
github.com/cockroachdb/cockroach/pkg/sql_test
testmain
0 runs so far, 0 failures, over 5s
0 runs so far, 0 failures, over 10s
0 runs so far, 0 failures, over 15s
0 runs so far, 0 failures, over 20s
0 runs so far, 0 failures, over 25s
0 runs so far, 0 failures, over 30s
0 runs so far, 0 failures, over 35s
0 runs so far, 0 failures, over 40s
0 runs so far, 0 failures, over 45s
0 runs so far, 0 failures, over 50s
0 runs so far, 0 failures, over 55s
0 runs so far, 0 failures, over 1m0s
0 runs so far, 0 failures, over 1m5s
0 runs so far, 0 failures, over 1m10s
0 runs so far, 0 failures, over 1m15s
0 runs so far, 0 failures, over 1m20s
0 runs so far, 0 failures, over 1m25s
0 runs so far, 0 failures, over 1m30s
0 runs so far, 0 failures, over 1m35s
0 runs so far, 0 failures, over 1m40s
0 runs so far, 0 failures, over 1m45s
0 runs so far, 0 failures, over 1m50s
0 runs so far, 0 failures, over 1m55s
0 runs so far, 0 failures, over 2m0s
0 runs so far, 0 failures, over 2m5s
0 runs so far, 0 failures, over 2m10s
0 runs so far, 0 failures, over 2m15s
0 runs so far, 0 failures, over 2m20s
0 runs so far, 0 failures, over 2m25s
0 runs so far, 0 failures, over 2m30s
0 runs so far, 0 failures, over 2m35s
0 runs so far, 0 failures, over 2m40s
0 runs so far, 0 failures, over 2m45s
0 runs so far, 0 failures, over 2m50s
0 runs so far, 0 failures, over 2m55s
0 runs so far, 0 failures, over 3m0s
0 runs so far, 0 failures, over 3m5s
0 runs so far, 0 failures, over 3m10s
0 runs so far, 0 failures, over 3m15s
0 runs so far, 0 failures, over 3m20s
0 runs so far, 0 failures, over 3m25s
0 runs so far, 0 failures, over 3m30s
0 runs so far, 0 failures, over 3m35s
0 runs so far, 0 failures, over 3m40s
0 runs so far, 0 failures, over 3m45s
0 runs so far, 0 failures, over 3m50s
0 runs so far, 0 failures, over 3m55s
0 runs so far, 0 failures, over 4m0s
0 runs so far, 0 failures, over 4m5s
0 runs so far, 0 failures, over 4m10s
0 runs so far, 0 failures, over 4m15s
0 runs so far, 0 failures, over 4m20s
0 runs so far, 0 failures, over 4m25s
0 runs so far, 0 failures, over 4m30s
0 runs so far, 0 failures, over 4m35s
0 runs so far, 0 failures, over 4m40s
0 runs so far, 0 failures, over 4m45s
0 runs so far, 0 failures, over 4m50s
0 runs so far, 0 failures, over 4m55s
0 runs so far, 0 failures, over 5m0s
0 runs so far, 0 failures, over 5m5s
0 runs so far, 0 failures, over 5m10s
0 runs so far, 0 failures, over 5m15s
0 runs so far, 0 failures, over 5m20s
0 runs so far, 0 failures, over 5m25s
0 runs so far, 0 failures, over 5m30s
0 runs so far, 0 failures, over 5m35s
0 runs so far, 0 failures, over 5m40s
0 runs so far, 0 failures, over 5m45s
0 runs so far, 0 failures, over 5m50s
0 runs so far, 0 failures, over 5m55s
0 runs so far, 0 failures, over 6m0s
0 runs so far, 0 failures, over 6m5s
0 runs so far, 0 failures, over 6m10s
0 runs so far, 0 failures, over 6m15s
0 runs so far, 0 failures, over 6m20s
0 runs so far, 0 failures, over 6m25s
0 runs so far, 0 failures, over 6m30s
0 runs so far, 0 failures, over 6m35s
0 runs so far, 0 failures, over 6m40s
0 runs so far, 0 failures, over 6m45s
0 runs so far, 0 failures, over 6m50s
0 runs so far, 0 failures, over 6m55s
0 runs so far, 0 failures, over 7m0s
0 runs so far, 0 failures, over 7m5s
0 runs so far, 0 failures, over 7m10s
0 runs so far, 0 failures, over 7m15s
0 runs so far, 0 failures, over 7m20s
0 runs so far, 0 failures, over 7m25s
0 runs so far, 0 failures, over 7m30s
0 runs so far, 0 failures, over 7m35s
0 runs so far, 0 failures, over 7m40s
0 runs so far, 0 failures, over 7m45s
0 runs so far, 0 failures, over 7m50s
0 runs so far, 0 failures, over 7m55s
0 runs so far, 0 failures, over 8m0s
0 runs so far, 0 failures, over 8m5s
0 runs so far, 0 failures, over 8m10s
0 runs so far, 0 failures, over 8m15s
0 runs so far, 0 failures, over 8m20s
0 runs so far, 0 failures, over 8m25s
0 runs so far, 0 failures, over 8m30s
0 runs so far, 0 failures, over 8m35s
0 runs so far, 0 failures, over 8m40s
0 runs so far, 0 failures, over 8m45s
0 runs so far, 0 failures, over 8m50s
0 runs so far, 0 failures, over 8m55s
0 runs so far, 0 failures, over 9m0s
0 runs so far, 0 failures, over 9m5s
0 runs so far, 0 failures, over 9m10s
0 runs so far, 0 failures, over 9m15s
0 runs so far, 0 failures, over 9m20s
0 runs so far, 0 failures, over 9m25s
0 runs so far, 0 failures, over 9m30s
0 runs so far, 0 failures, over 9m35s
fatal error: runtime: out of memory

runtime stack:
runtime.throw(0x503d67, 0x16)
	/usr/local/go/src/runtime/panic.go:566 +0x95
runtime.sysMap(0xc421600000, 0x200000, 0x0, 0x5a1d78)
	/usr/local/go/src/runtime/mem_linux.go:219 +0x1d0
runtime.(*mheap).sysAlloc(0x589520, 0x200000, 0x40e500)
	/usr/local/go/src/runtime/malloc.go:407 +0x37a
runtime.(*mheap).grow(0x589520, 0x100, 0x0)
	/usr/local/go/src/runtime/mheap.go:726 +0x62
runtime.(*mheap).allocSpanLocked(0x589520, 0x100, 0x1)
	/usr/local/go/src/runtime/mheap.go:630 +0x4f2
runtime.(*mheap).alloc_m(0x589520, 0x100, 0x100000000, 0x425ef6)
	/usr/local/go/src/runtime/mheap.go:515 +0xe0
runtime.(*mheap).alloc.func1()
	/usr/local/go/src/runtime/mheap.go:579 +0x4b
runtime.systemstack(0xc420187ed0)
	/usr/local/go/src/runtime/asm_amd64.s:314 +0xab
runtime.(*mheap).alloc(0x589520, 0x100, 0x10100000000, 0xc42019a030)
	/usr/local/go/src/runtime/mheap.go:580 +0x73
runtime.largeAlloc(0x1ffe00, 0xc42019a001, 0xc4201ebd70)
	/usr/local/go/src/runtime/malloc.go:774 +0x93
runtime.mallocgc.func1()
	/usr/local/go/src/runtime/malloc.go:669 +0x3e
runtime.systemstack(0xc42001f500)
	/usr/local/go/src/runtime/asm_amd64.s:298 +0x79
runtime.mstart()
	/usr/local/go/src/runtime/proc.go:1079

goroutine 51 [running]:
runtime.systemstack_switch()
	/usr/local/go/src/runtime/asm_amd64.s:252 fp=0xc4201ebc78 sp=0xc4201ebc70
runtime.mallocgc(0x1ffe00, 0x4db120, 0xc4201ebd01, 0x44e4a0)
	/usr/local/go/src/runtime/malloc.go:670 +0x903 fp=0xc4201ebd18 sp=0xc4201ebc78
runtime.makeslice(0x4db120, 0x1ffe00, 0x1ffe00, 0x0, 0x0, 0xffe00)
	/usr/local/go/src/runtime/slice.go:57 +0x7b fp=0xc4201ebd70 sp=0xc4201ebd18
bytes.makeSlice(0x1ffe00, 0x0, 0x0, 0x0)
	/usr/local/go/src/bytes/buffer.go:198 +0x77 fp=0xc4201ebdb0 sp=0xc4201ebd70
bytes.(*Buffer).ReadFrom(0xc420114000, 0x572340, 0xc420128010, 0xc420031688, 0xc420031601, 0xc420031688)
	/usr/local/go/src/bytes/buffer.go:170 +0x2a9 fp=0xc4201ebe20 sp=0xc4201ebdb0
io.copyBuffer(0x572280, 0xc420114000, 0x572340, 0xc420128010, 0x0, 0x0, 0x0, 0xc4200f2000, 0x5, 0xc4200f2000)
	/usr/local/go/src/io/io.go:384 +0x323 fp=0xc4201ebea8 sp=0xc4201ebe20
io.Copy(0x572280, 0xc420114000, 0x572340, 0xc420128010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68 fp=0xc4201ebf08 sp=0xc4201ebea8
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d fp=0xc4201ebf68 sp=0xc4201ebf08
os/exec.(*Cmd).Start.func1(0xc4200f2000, 0xc420132080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27 fp=0xc4201ebfa0 sp=0xc4201ebf68
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1 fp=0xc4201ebfa8 sp=0xc4201ebfa0
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 1 [select]:
main.main()
	/go/src/github.com/cockroachdb/stress/main.go:108 +0xa86

goroutine 5 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6e5, 0xc4201d2000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc420170d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc42019a120, 0x50eb78, 0xc420190000, 0xc420198080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc42019a120, 0x0, 0x40fa3e, 0xc420196028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc42019a120, 0xc420170e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc420190000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc420190000, 0xc420192000, 0xc42002fee0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc420190000, 0x50eb58, 0xc42018e060, 0xc420190000, 0x5, 0xc420190000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 6 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6eb, 0xc420226000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc420174d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc420206120, 0x50eb78, 0xc420154000, 0xc420204080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc420206120, 0x0, 0x40fa3e, 0xc420182028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc420206120, 0xc420174e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc420154000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc420154000, 0xc42016c000, 0xc4200306e0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc420154000, 0x50eb58, 0xc42014e060, 0xc420154000, 0x5, 0xc420154000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 7 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6da, 0xc420116000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc420046d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc4200d2120, 0x50eb78, 0xc420098000, 0xc4200be080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc4200d2120, 0x0, 0x40fa3e, 0xc4200bc028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc4200d2120, 0xc420046e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc420098000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc420098000, 0xc4200ae000, 0xc420030ee0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc420098000, 0x50eb58, 0xc420094060, 0xc420098000, 0x5, 0xc420098000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 8 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6e1, 0xc420194080, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc4200c0d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc42013e150, 0x50eb78, 0xc4200f2000, 0xc420132080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc42013e150, 0x0, 0x40fa3e, 0xc420128028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc42013e150, 0xc4200c0e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc4200f2000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc4200f2000, 0xc420114000, 0xc4200316e0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc4200f2000, 0x50eb58, 0xc4200ee060, 0xc4200f2000, 0x5, 0xc4200f2000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 9 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6e3, 0xc420214000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc4200c6d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc4200f0210, 0x50eb78, 0xc4200b4160, 0xc4200e82c0)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc4200f0210, 0x0, 0x40fa3e, 0xc4200e4050)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc4200f0210, 0xc4200c6e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc4200b4160, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc4200b4160, 0xc4200b8070, 0xc420031ee0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc4200b4160, 0x50eb58, 0xc4200b21e0, 0xc4200b4160, 0x5, 0xc4200b4160)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 10 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6dc, 0xc420194000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc4200c4d00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc42013e120, 0x50eb78, 0xc4200b4000, 0xc4200e8080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc42013e120, 0x0, 0x40fa3e, 0xc4200e4028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc42013e120, 0xc4200c4e08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc4200b4000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc4200b4000, 0xc4200b8000, 0xc42002a6e0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc4200b4000, 0x50eb58, 0xc4200b2060, 0xc4200b4000, 0x5, 0xc4200b4000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 11 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6d8, 0xc420090100, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc42004ad00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc420010480, 0x50eb78, 0xc4200a2000, 0xc42000e200)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc420010480, 0x0, 0x40fa3e, 0xc420032068)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc420010480, 0xc42004ae08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc4200a2000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc4200a2000, 0xc420056070, 0xc42002aee0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc4200a2000, 0x50eb58, 0xc42001a240, 0xc4200a2000, 0x5, 0xc4200a2000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 12 [syscall, 9 minutes]:
syscall.Syscall6(0xf7, 0x1, 0x6e9, 0xc420200000, 0x1000004, 0x0, 0x0, 0x4e4f80, 0xc4201ead00, 0x4316be)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc4201f2120, 0x50eb78, 0xc420162000, 0xc4201f0080)
	/usr/local/go/src/os/wait_waitid.go:28 +0xbc
os.(*Process).wait(0xc4201f2120, 0x0, 0x40fa3e, 0xc4201ee028)
	/usr/local/go/src/os/exec_unix.go:22 +0xab
os.(*Process).Wait(0xc4201f2120, 0xc4201eae08, 0x0, 0x0)
	/usr/local/go/src/os/doc.go:49 +0x2b
os/exec.(*Cmd).Wait(0xc420162000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:434 +0x6d
os/exec.(*Cmd).Run(0xc420162000, 0xc4201e4000, 0xc42002b6e0)
	/usr/local/go/src/os/exec/exec.go:279 +0x48
os/exec.(*Cmd).CombinedOutput(0xc420162000, 0x50eb58, 0xc42011c060, 0xc420162000, 0x5, 0xc420162000)
	/usr/local/go/src/os/exec/exec.go:494 +0x140
main.main.func1(0xc420012430, 0xc420032028, 0xc420032030, 0xc42001a120)
	/go/src/github.com/cockroachdb/stress/main.go:94 +0x10d
created by main.main
	/go/src/github.com/cockroachdb/stress/main.go:103 +0x258

goroutine 14 [select, 9 minutes]:
main.main.func1.1(0xc42001a240, 0xc4200a2000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 17 [select, 9 minutes]:
main.main.func1.1(0xc420094060, 0xc420098000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 33 [select, 9 minutes]:
main.main.func1.1(0xc4200b2060, 0xc4200b4000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 49 [select, 9 minutes]:
main.main.func1.1(0xc4200ee060, 0xc4200f2000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 18 [syscall]:
syscall.Syscall(0x0, 0x9, 0xc420345bef, 0xea211, 0x6b, 0xea27c, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0x9, 0xc420345bef, 0xea211, 0xea211, 0x6b, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0x9, 0xc420345bef, 0xea211, 0xea211, 0x6b, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc4200bc010, 0xc420345bef, 0xea211, 0xea211, 0x6b, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc4200bc010, 0xc420345bef, 0xea211, 0xea211, 0x6b, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc4200ae000, 0x572340, 0xc4200bc010, 0xc420030e88, 0xc420030e01, 0xc420030e88)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc4200ae000, 0x572340, 0xc4200bc010, 0x0, 0x0, 0x0, 0xc420098000, 0x5, 0xc420098000)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc4200ae000, 0x572340, 0xc4200bc010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc420098000, 0xc4200be080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 15 [syscall]:
syscall.Syscall(0x0, 0x4, 0xc421467bff, 0xf0201, 0x469, 0xf066a, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0x4, 0xc421467bff, 0xf0201, 0xf0201, 0x469, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0x4, 0xc421467bff, 0xf0201, 0xf0201, 0x469, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc420032050, 0xc421467bff, 0xf0201, 0xf0201, 0x469, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc420032050, 0xc421467bff, 0xf0201, 0xf0201, 0x469, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc420056070, 0x572340, 0xc420032050, 0xc42002ae88, 0xc42002ae01, 0xc42002ae88)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc420056070, 0x572340, 0xc420032050, 0x0, 0x0, 0x0, 0xc4200a2000, 0x5, 0xc4200a2000)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc420056070, 0x572340, 0xc420032050, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc4200a2000, 0xc42000e200)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 65 [select, 9 minutes]:
main.main.func1.1(0xc42014e060, 0xc420154000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 81 [select, 9 minutes]:
main.main.func1.1(0xc42011c060, 0xc420162000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 97 [select, 9 minutes]:
main.main.func1.1(0xc42018e060, 0xc420190000)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 50 [syscall]:
syscall.Syscall(0x0, 0xd, 0xc420861606, 0xf67fa, 0x8a7, 0xf70a1, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0xd, 0xc420861606, 0xf67fa, 0xf67fa, 0x8a7, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0xd, 0xc420861606, 0xf67fa, 0xf67fa, 0x8a7, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc4200e4010, 0xc420861606, 0xf67fa, 0xf67fa, 0x8a7, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc4200e4010, 0xc420861606, 0xf67fa, 0xf67fa, 0x8a7, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc4200b8000, 0x572340, 0xc4200e4010, 0xc4200f8688, 0x1, 0x0)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc4200b8000, 0x572340, 0xc4200e4010, 0x0, 0x0, 0x0, 0xc4200ee120, 0x4021eb, 0x180001)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc4200b8000, 0x572340, 0xc4200e4010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc4200b4000, 0xc4200e8080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 34 [select, 9 minutes]:
main.main.func1.1(0xc4200b21e0, 0xc4200b4160)
	/go/src/github.com/cockroachdb/stress/main.go:76 +0x309
created by main.main.func1
	/go/src/github.com/cockroachdb/stress/main.go:92 +0x3df

goroutine 98 [syscall]:
syscall.Syscall(0x0, 0xf, 0xc4210683b7, 0xefa49, 0x6ab, 0xf00f4, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0xf, 0xc4210683b7, 0xefa49, 0xefa49, 0x6ab, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0xf, 0xc4210683b7, 0xefa49, 0xefa49, 0x6ab, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc420196010, 0xc4210683b7, 0xefa49, 0xefa49, 0x6ab, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc420196010, 0xc4210683b7, 0xefa49, 0xefa49, 0x6ab, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc420192000, 0x572340, 0xc420196010, 0xc42002fe88, 0xc42002fe01, 0xc42002fe88)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc420192000, 0x572340, 0xc420196010, 0x0, 0x0, 0x0, 0xc420190000, 0x5, 0xc420190000)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc420192000, 0x572340, 0xc420196010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc420190000, 0xc420198080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 82 [syscall]:
syscall.Syscall(0x0, 0x7, 0xc42126782b, 0xf05d5, 0x469, 0xf0a3e, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0x7, 0xc42126782b, 0xf05d5, 0xf05d5, 0x469, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0x7, 0xc42126782b, 0xf05d5, 0xf05d5, 0x469, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc4201ee010, 0xc42126782b, 0xf05d5, 0xf05d5, 0x469, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc4201ee010, 0xc42126782b, 0xf05d5, 0xf05d5, 0x469, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc4201e4000, 0x572340, 0xc4201ee010, 0xc42002b688, 0xc42002b601, 0xc42002b688)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc4201e4000, 0x572340, 0xc4201ee010, 0x0, 0x0, 0x0, 0xc420162000, 0x5, 0xc420162000)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc4201e4000, 0x572340, 0xc4201ee010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc420162000, 0xc4201f0080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 35 [syscall]:
syscall.Syscall(0x0, 0xb, 0xc420664857, 0xf35a9, 0x96f, 0xf3f18, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0xb, 0xc420664857, 0xf35a9, 0xf35a9, 0x96f, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0xb, 0xc420664857, 0xf35a9, 0xf35a9, 0x96f, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc4200e4038, 0xc420664857, 0xf35a9, 0xf35a9, 0x96f, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc4200e4038, 0xc420664857, 0xf35a9, 0xf35a9, 0x96f, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc4200b8070, 0x572340, 0xc4200e4038, 0xc42009e688, 0x1, 0x0)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc4200b8070, 0x572340, 0xc4200e4038, 0x0, 0x0, 0x0, 0xc4200b2360, 0x4021eb, 0x180001)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc4200b8070, 0x572340, 0xc4200e4038, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc4200b4160, 0xc4200e82c0)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db

goroutine 66 [syscall]:
syscall.Syscall(0x0, 0x5, 0xc420e69831, 0xee5cf, 0x46a, 0xeea39, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0x5, 0xc420e69831, 0xee5cf, 0xee5cf, 0x46a, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x55
syscall.Read(0x5, 0xc420e69831, 0xee5cf, 0xee5cf, 0x46a, 0x0, 0x0)
	/usr/local/go/src/syscall/syscall_unix.go:161 +0x49
os.(*File).read(0xc420182010, 0xc420e69831, 0xee5cf, 0xee5cf, 0x46a, 0x0, 0x0)
	/usr/local/go/src/os/file_unix.go:228 +0x4f
os.(*File).Read(0xc420182010, 0xc420e69831, 0xee5cf, 0xee5cf, 0x46a, 0x0, 0x0)
	/usr/local/go/src/os/file.go:101 +0x59
bytes.(*Buffer).ReadFrom(0xc42016c000, 0x572340, 0xc420182010, 0xc420030688, 0xc420030601, 0xc420030688)
	/usr/local/go/src/bytes/buffer.go:176 +0x155
io.copyBuffer(0x572280, 0xc42016c000, 0x572340, 0xc420182010, 0x0, 0x0, 0x0, 0xc420154000, 0x5, 0xc420154000)
	/usr/local/go/src/io/io.go:384 +0x323
io.Copy(0x572280, 0xc42016c000, 0x572340, 0xc420182010, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:360 +0x68
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:253 +0x4d
os/exec.(*Cmd).Start.func1(0xc420154000, 0xc420204080)
	/usr/local/go/src/os/exec/exec.go:370 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:371 +0x4db
Makefile:138: recipe for target 'stress' failed
make: *** [stress] Error 2

```

--- a/pkg/cmd/teamcity-trigger/main.go
+++ b/pkg/cmd/teamcity-trigger/main.go
@@ -22,6 +22,7 @@ import (
 	"net/url"
 	"os"
 	"strconv"
+	"strings"
 
 	"github.com/abourget/teamcity"
 	"github.com/kisielk/gotool"
@@ -33,6 +34,9 @@ var branchName = flag.String("branch", "", "the VCS branch to build")
 const teamcityServerURLEnv = "TC_SERVER_URL"
 const teamcityAPIUserEnv = "TC_API_USER"
 const teamcityAPIPasswordEnv = "TC_API_PASSWORD"
+
+const tagsKey = "env.TAGS"
+const stressTag = "stress"
 
 func main() {
 	flag.Parse()
@@ -61,11 +65,16 @@ func main() {
 		for _, params := range []map[string]string{
 			{}, // uninstrumented
 			{"env.GOFLAGS": "-race"},
-			{"env.TAGS": "deadlock"},
+			{tagsKey: "deadlock"},
 		} {
 			for _, importPath := range importPaths {
 				params["env.PKG"] = importPath
 				params["env.COCKROACH_PROPOSER_EVALUATED_KV"] = strconv.FormatBool(propEvalKV)
+				if tags, ok := params[tagsKey]; ok {
+					params[tagsKey] = strings.Join([]string{tags, stressTag}, " ")
+				} else {
+					params[tagsKey] = stressTag
+				}
 				build, err := client.QueueBuild(*buildTypeID, *branchName, params)
 				if err != nil {
 					log.Fatalf("failed to create teamcity build (*buildTypeID=%s *branchName=%s, params=%+v): %s", *buildTypeID, *branchName, params, err)

--- a/pkg/sql/logic_gc_test.go
+++ b/pkg/sql/logic_gc_test.go
@@ -14,7 +14,7 @@
 //
 // Author: Tamir Duberstein (tamird@gmail.com)
 
-// +build gc
+// +build !stress,gc
 
 package sql_test
 

--- a/pkg/sql/logic_not_gc_test.go
+++ b/pkg/sql/logic_not_gc_test.go
@@ -14,7 +14,7 @@
 //
 // Author: Tamir Duberstein (tamird@gmail.com)
 
-// +build !gc
+// +build !stress,!gc
 
 package sql_test
 

--- a/pkg/sql/logic_test.go
+++ b/pkg/sql/logic_test.go
@@ -14,6 +14,8 @@
 //
 // Author: Peter Mattis (peter@cockroachlabs.com)
 
+// +build !stress
+
 package sql_test
 
 import (

--- a/pkg/sql/parallel_test.go
+++ b/pkg/sql/parallel_test.go
@@ -21,6 +21,8 @@
 // dir contains a "test.yaml" file along with a set of files in logic test
 // format. The test.yaml file corresponds to the parTestSpec structure below.
 
+// +build !stress
+
 package sql_test
 
 import (

